### PR TITLE
docs: Improve readme

### PR DESCRIPTION
**Pull request by ebebbington**
What do you guys think? @lucacasonato @qu4k (be sure to open the branch on GH to see how it looks)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,10 +84,10 @@
 
 - [bson](https://github.com/mongodb/bson-rust) - rust bson crate
 
-### Contribution
+## Contribution
```
Comment (notfilippo) [inline README.md]: Revert, as this is part of `## other` section

File: README.md [hunk 2]
```diff
@@ -27,10 +84,10 @@
...
 
 Pull request, issues and feedback are very welcome. Code style is formatted with `deno fmt` and `cargo fmt` and commit messages are done following Conventional Commits spec.
 
-### Licence
+## Licence
```
Comment (notfilippo) [inline README.md]: Ditto.

File: README.md [hunk 3]
```diff
@@ -2,19 +2,76 @@
     <img align="right" src="assets/mango.png" height="150" />
 </span>
 
-# mango
+<p align="center">
+  <img height="200" src="assets/mango.png" alt="Mango Logo">
+  <h1 align="center">Mango</h1>
+</p>
```
Comment (notfilippo) [inline README.md:4]: Now there are two mango 🥭

**Review by notfilippo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ebebbington) [reply README.md]: My bad, yeah that's a typo on my part

File: README.md (hunk 2 above)
Comment (ebebbington) [reply README.md]: Same as above 👍

File: README.md (hunk 1 above)
Comment (ebebbington) [reply README.md]: https://github.com/denodrivers/mango/pull/15/commits/5e0e3d9d15d235bb4aca644f1f3e65e2bcd200d9

File: README.md (hunk 2 above)
Comment (ebebbington) [reply README.md]: https://github.com/denodrivers/mango/pull/15/commits/5e0e3d9d15d235bb4aca644f1f3e65e2bcd200d9

File: README.md (hunk 3 above)
Comment (ebebbington) [reply README.md:4]: https://github.com/denodrivers/mango/pull/15/commits/5e0e3d9d15d235bb4aca644f1f3e65e2bcd200d9

**Review by notfilippo — APPROVED** [review]
LGTM 🥭

**Comment by ebebbington** [comment]
Cool, cheers @qu4k and thanks for catching those!

**Pull request merged**